.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: [ 'release', 'releases/**']
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to build'
+        required: true
+        default: 'main'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
```

Comment:
Is this the image we have been using?

Reply:
this is an GH Action thing. It does not effect the build container.